listing-description/en-US.md

```diff
@@ -2,39 +2,35 @@
 
 A Fabric-based Minecraft modpack focusing on performance and graphics enhancements.
 
-[![Current review](https://img.youtube.com/vi/bb8G9X5Q_4I/hqdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
+[![Video review](https://img.youtube.com/vi/bb8G9X5Q_4I/hqdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
 
 ### Performance
 
-Fabulously Optimized is usually faster than OptiFine thanks to [modern performance mods][1] such as Sodium, Lithium and more!
+Fabulously Optimized is faster than OptiFine thanks to [modern performance mods][1] such as Sodium, Lithium and others! The bundled mods have been carefully selected to be safe to your worlds and bring noticeable speed benefits on any device.
 
 ### OptiFine parity
 
-Fabulously Optimized [supports almost all OptiFine features][2], including shaders, connected textures and zoom. [You can even get your own free cape!][3]
+Fabulously Optimized [supports most OptiFine features][2], including shaders, connected textures and zoom. If your resource pack says "OptiFine required", it will likely work on here as well. [You can also get a free cape!][3]
```

Comment:
```suggestion
Fabulously Optimized [offers support for most OptiFine features][2], including shaders, connected textures, and zoom. If your resource pack states "OptiFine required," it will probably function here as well. [Additionally, you can get your own free cape.][3]!
```

Reply:
I don't "offer support" like providing help or having add-ons, I already "have the support". Also, reverted the cape sentence.